### PR TITLE
Fix statistics compatibility

### PR DESCRIPTION
**Pull request by Novemser**
Make statistic compatible with v1.0 - v2.0

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: '2.3'
 
 services:
   pd0:
-    image: pingcap/pd:latest
+    image: pingcap/pd:v1.0.1
```
Comment (ilovesoup) [inline docker-compose.yaml]: Why is this?

Comment (Novemser) [reply docker-compose.yaml]: Just for testing lower version of TiDB, will change this later.

**Comment by Novemser** [comment]
/run-all-tests tikv=release-1.1 tidb=release-1.1 pd=release-1.1

File: docker-compose.yaml (hunk 1 above)
Comment (Novemser) [reply docker-compose.yaml]: DONE

**Comment by Novemser** [comment]
/run-all-tests tikv=release-1.1 tidb=release-1.1 pd=release-1.1
--

**Comment by Novemser** [comment]
/run-all-tests tikv=release-1.1 tidb=release-1.1 pd=release-1.1

**Comment by Novemser** [comment]
/run-all-tests tikv=release-1.1 tidb=release-1.1 pd=release-1.1

**Comment by Novemser** [comment]
/run-all-tests tikv=release-1.0 tidb=release-1.0 pd=release-1.0

**Comment by Novemser** [comment]
v1.0 failed due to TiKV overflow, nothing relevant to statistic information.

**Comment by Novemser** [comment]
/run-all-tests tikv=release-1.1 tidb=release-1.1 pd=release-1.1

**Pull request merged**